README.md

```diff
@@ -24,7 +25,8 @@ Table of contents
       * [6. License](#5-license)
       * [7. Acknowledgments](#6-acknowledgments)
 
-
+# Requirements
+Please use google provider version = "~> 2.3"
```

Comment:
Please add information that without specifying min_master_version and version in node pools, GKE is using actually default version which might change in near future.